modules/end-user-guide/pages/devfile-introduction.adoc

```diff
@@ -5,21 +5,24 @@ link:https://devfile.io/[Devfiles] are `yaml` text files used for development en
 
 .{prod}-specific devfile features
 
+[NOTE]
+====    
+{prod} is expected to work with most of the popular images defined in the `components` section of devfile. For production purposes, it is recommended to use one of the link:https://catalog.redhat.com/software/containers/search?gs&q=ubi[Universal Base Images] as a base image for defining the Cloud Development Environment.
```

Comment:
some ubi images can not be used: 
- `ubi8-9` https://github.com/eclipse-che/che/issues/22870
- ubi7-based images where not tested

Reply:
I don't know if this ^ should be mentioned, just letting you know about this limitation...